alma/alma-8.x/common/install_utils.sh

```diff
@@ -53,6 +52,11 @@ yum install -y numactl \
     gcc-gfortran \
     perl
 
+# Install environment-modules 5.0.1
+wget https://repo.almalinux.org/vault/9.1/BaseOS/x86_64/os/Packages/environment-modules-5.0.1-1.el9.x86_64.rpm
```

Comment:
Is the compatibility tested? We currently only support AlmaLinux HPC 8.7 and the vault only shows support for 4.5.2

Reply:
I tried to install it and run it on AlmaLinux HPC 8.7 and I did not encounter any issues